lib/rubocop/cop/rspec/file_path.rb

```diff
@@ -97,11 +95,13 @@ def ensure_correct_file_path(send_node, example_group, arguments)
         end
 
         def routing_spec?(args)
-          args.any?(&method(:routing_metadata?))
+          args.any?(&method(:routing_metadata?)) || routing_spec_path?
```

Comment:
I've removed this, and there were no differences between master/PR/PR with removed `routing_metadata?` runs against real-world-rspec.

```suggestion
          routing_spec_path?
```

Reply:
If there is a test file like spec/routes/foo_spec.rb including `type: :routing` to indicate that it is a routing spec, it would fail by this change. But you don't think there are any such tests in real application, so that means it should be ok to remove this check, right?